_posts/2022-05-18-solving-zdeveau-21-with-radare2.md

```diff
@@ -0,0 +1,184 @@
+---
+# Editing the below variables is the absolute minimum to get stuff up. Feel
+# free [obliged] to add editorial niceties or strip the page if you need to
+# include something more complex.
+season: 19
+episode: 4
+layout: default
+title: NorthSec CTF 2021's Rare Metal Sequence
+# summary for social media cards. Ideal length 55 chars, max 200
+summary: Reverse a SNES crackme by Zack Deveau using radare2 and Jupyter Notebooks
+tags: reverse-engineering
+date_fr: 18 mai 2022
+date_long_fr: mercredi le 18 mai 2022
+date_en: Mai 18th 2022
+date_long_en: Wednesday May 18th 2022
+description_fr: >
+   Les NSec 2021 CTF challenges "Rare metal sequencer" 1 et 2 etait des crackme par Zack Deveau. Zack a crée les challenge pour le Super Nintendo Entertainment System (SNES) et let participants devaient le reverser pour trouver les entrées du joypad qui vas le débloquer. Nous alons etudié comment utiliser radare2 pour le reverser en deux directions: en l'avant et en l'arrière. Aussi en prime nous allons montrer comment on peut fair le 2022 Nsec warmup challenge logically bonus flag.
```

Comment:
```suggestion
   Le challenge du CTF NSec 2021 "Rare metal sequencer" 1 et 2 sont des crackme par Zack Deveau. Zack a créé les challenges pour le Super Nintendo Entertainment System (SNES) et les participants devaient faire de la rétro-ingénierie pour trouver les entrées du joypad pour résoudre le défi. Nous allons voir comment utiliser radare2 pour le *reverse* en deux directions: par l'avant et par l'arrière (forward and backward). En prime, nous allons montrer comment on peut résoudre le défi d'échauffement du CTF du NSec 2022 *logically bonus flag*.
```

Reply:
thank you for fixing my attempt at the french description!